src/Symfony/Component/Translation/Command/TranslationPushCommand.php

```diff
@@ -21,10 +21,12 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Translation\Event\TranslationPushEvent;
 use Symfony\Component\Translation\Provider\FilteringProvider;
 use Symfony\Component\Translation\Provider\TranslationProviderCollection;
 use Symfony\Component\Translation\Reader\TranslationReaderInterface;
 use Symfony\Component\Translation\TranslatorBag;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
```

Comment:
As this only dispatches an event, it should depend on the PSR interface instead to be more flexible.